src/client/sandbox/node/window.js

```diff
@@ -222,6 +222,16 @@ export default class WindowSandbox extends SandboxBase {
         };
         window.Image.prototype = nativeMethods.Image.prototype;
 
+        window.Function = function (...args) {
+            var lastArgIndex = args.length - 1;
+
+            if (typeof args[lastArgIndex] === 'string')
+                args[lastArgIndex] = processScript(args[lastArgIndex], false, false);
+
+            return nativeMethods.Function.apply(this, args);
```

Comment:
Any of window's classes don't check. If you think that it needs, you can make separate issue to check all classes